Add direct reference to migration status in instructions for arch migration (#1895)
conda-forge/conda-forge.github.io: src/maintainer/knowledge_base.rst

To request a migration for a particular package and all its dependencies:

1. It may be that your package is already in the process of being migrated. Please check
   the status of the

>    `arm osx addition Migration Status <https://conda-forge.org/status/#armosxaddition>`_.

Reviewer: ```suggestion
   `arm osx addition Migration Status <https://conda-forge.org/status/#armosxaddition>`__.
```

   If your package is already in the process of being migrated, it will appear
   under the appropriate heading (done, in-pr, awaiting-parents, etc.).
2. Check the feedstock in question to see if there is already an issue or pull request.
   Opening an issue here is fine, as it might take a couple iterations of the below,
   especially if many dependencies need to be built as well.
3. If nothing is under way, look at the current `conda-forge-pinning <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/osx_arm64.txt>`__.
4. If the package is not listed there, make a PR, adding the package
   name to a random location in ``osx_arm64.txt``.
   The migration bot should start making automated pull requests to the
   repo and its dependencies.

> 5. Within a few hours, the `status page <https://conda-forge.org/status/#armosxaddition>`_

Reviewer: ```suggestion
5. Within a few hours, the `status page <https://conda-forge.org/status/#armosxaddition>`__
```

Using ``arch_rebuild.txt``
==========================

You can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with
different architectures/platforms (such as ppc64le or aarch64).
Check the

> `migration status <https://conda-forge.org/status/#aarch64andppc64leaddition>`_

Reviewer: ```suggestion
`migration status <https://conda-forge.org/status/#aarch64andppc64leaddition>`__
```

to see if your package is already in the queue to get migrated. If not, you can

> add the feedstock to ``arch_rebuild.txt``, open a PR to the

Reviewer: ```suggestion
add the feedstock to ``arch_rebuild.txt`` by opening a PR to the
```
